main/Kconfig.projbuild

```diff
@@ -10,13 +10,13 @@ config WIFI_PASSWORD
     string "WiFi Password"
     default "mypassword"
     help
-	WiFi password (WPA or WPA2) for the example to use.
+    WiFi password (WPA or WPA2) for the example to use.
```

Comment:
Can you revert the indentation changes, that do not modify the content, e.g. this line?